__tests__/app.test.js

```diff
@@ -93,9 +93,47 @@ describe('GET /api/articles/:article_id', () => {
           .expect(400)
           .then((response) => {
             expect(response.body.message).toBe("Invalid input syntax");
-          });
-      });
+        });
+    });
+});
+
+//5
+describe.only('GET /api/articles responds with array of articles', () => {
+    test('GET:200 sends an appropriate status and article array', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({body}) => {
+            
+            expect(body.articles).toHaveLength(13)
 
+            body.articles.forEach((article) => {
+                expect(typeof(article.article_id)).toBe('number')
+                expect(typeof(article.title)).toBe('string')
+                expect(typeof(article.author )).toBe('string')
+                expect(typeof(article.topic )).toBe('string')
+                expect(typeof(article.created_at )).toBe('string')
+                expect(typeof(article.votes)).toBe('number')
+                expect(typeof(article.article_img_url)).toBe('string')
+                expect(typeof(article.comment_count)).toBe('string')
+            })
+        })
+    });
+    test('articles should be ordered in descending order by created_at',  () => {
+        return request(app)
+        .get('/api/articles')
+        .then(({body}) => {      
+            expect(body.articles).toBeSortedBy('created_at', {descending: true})
+        })
+    });
 
+    test('should return 404 if articles spelt wrong', () =>{
+        return request(app)
+        .get('/api/articles-spelt-wrong')
+        .expect(404)
+        .then((response) => {
+            expect(response.body.message).toBe('path is not found')
+        })
```

Comment:
This test isn't necessary. The earlier bad path test at the beginning of this file already proves this error is handled